demos/tool_dataset_splitting_by_language/dataset_splitting_by_language.py

```diff
@@ -0,0 +1,91 @@
+# This tool is used to split datasets to sub-datasets
+# by fast-text lanuage model.
+
+import os
+
+import fire
+import pandas as pd
+from jsonargparse import Namespace
+from loguru import logger
+
+from data_juicer.format import load_formatter
+from data_juicer.ops.filter.language_id_score_filter import \
+    LanguageIDScoreFilter
+
+
+def keep_by_lang(sample, lang):
+    """
+    Keep samples with the specified language.
+    :param sample: a sample in dataset
+    :param lang: the specified language
+    :return: True to keep,  False to discard
+    """
+    if sample['stats']['lang'] == lang:
+        return True
+    return False
+
+
+def main(src_dir,
+         target_dir,
+         text_keys_to_load=None,
+         text_key_to_process='text',
+         suffixes=[],
+         num_proc=1):
+    """
+    Load dataset from the source directory, then apply language identification
+    using the operation filter called `LanguageIDScoreFilter`,
+    finally, split the dataset by language and save it.
+    :param src_dir: path thats store dataset directory
```

Comment:
path thats store dataset directory -> path to store the dataset

Reply:
done